### DISCUSSION TITLE
Welcome to WernhersOldStuff Discussions!

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/WernhersOldStuff#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AtomikkuSan** [comment]
So, I've already begun to make a new, coherent theme of how we should name and describe the parts. Any suggestions?

**Reply by zer0Kerbal** [reply]
IF change PART{name} then we need to supply a `ghostPart.cfg` so changing PART{name} doesn't wreck save games (or maybe shouldn't change PART{name}'s)

* all lower case
* all filenames should equal part names `wos-xxx-xxxx-01` and `wos-xxx-xxxx-01.cfg`
* all part names start with `wos`

I was going to get to that in the next 24-36 hours so I can run my auto-part localizer on all of them.
